dist/pythonlibs/testrunner/spawn.py

```diff
@@ -39,7 +39,13 @@
 # By default never reset after the terminal is open unless explicitly requested
 # through an environment variable.
 TESTRUNNER_RESET_AFTER_TERM = int(os.environ.get('TESTRUNNER_RESET_AFTER_TERM')
-                                  or '0')
+                                  or 0)
+
+# When running e.g. tests/shell_ble we don't want to reset the board, because
+# then ble-serial would terminate and the created virtual serial port would get
+# lost. By default the board is reset before the test starts.
+TESTRUNNER_RESET_BOARD_ON_STARTUP = \
+    int(os.environ.get('TESTRUNNER_RESET_BOARD_ON_STARTUP') or 1)
```

Comment:
```suggestion
    int(os.environ.get('TESTRUNNER_RESET_BOARD_ON_STARTUP') or '1')
```

Reply:
As mentioned above, I think that it should be simply the number by itself and not the string representation of it.

Reply:
ok doesn't matter